runtime/startup_aux.c

```diff
@@ -111,8 +116,7 @@ void caml_parse_ocamlrunparam(void)
     while (*opt != '\0'){
       switch (*opt++){
       case 'a': scanmult (opt, &caml_init_policy); break;
-      case 'b': scanmult (opt, &p); caml_record_backtrace(Val_int (p));
-        break;
+      case 'b': scanmult (opt, &p); record_backtrace = p; break;
```

Comment:
Given the other cases, this looks like it could be `scanmult (opt, &record_backtrace);`?

Reply:
thanks for catching that!